tutorials/assets_pipeline/importing_scenes.rst

```diff
@@ -188,327 +188,538 @@ the scene will lead to more consistent results and should be done whenever
+scene in the FileSystem dock:
+
+- **Root Type:** The node type to use as a root node. Using node types that
+  inherit from Node3D is recommended. Otherwise, you'll lose the ability to
+  position the node directly in the 3D editor.
+- **Root Name:** The name of the root node in the imported scene. This is
+  generally not noticeable when instancing the scene in the editor (or
+  drag-and-dropping from the FileSystem dock), as the root node is renamed to
+  match the filename in this case.
+- **Apply Root Scale:** If enabled, **Root Scale** will be *applied* on the
+  meshes and animations directly, while keeping the root node's scale to the
+  default `(1, 1, 1)`. This means that if you add a child node later on within
+  the imported scene, it won't be scaled. If disabled, **Root Scale** will
+  multiply the scale of the root node instead.
+
+**Meshes**
+
+- **Ensure Tangents:** If checked, generate vertex tangents using
+  `Mikktspace <http://www.mikktspace.com/>`__ if the input meshes don't have
+  tangent data. When possible, it's recommended to let the 3D modeling software
+  generate tangents on export instead on relying on this option. Tangents are
+  required for correct display of normal and height maps, along with any
+  material/shader features that require tangents. If you don't need material
+  features that require tangents, disabling this can reduce output file size and
+  speed up importing if the source 3D file doesn't contain tangents.
+- **Generate LODs:** If checked, generates lower detail variants of the
+  mesh which will be displayed in the distance to improve rendering performance.
+  Not all meshes benefit from LOD, especially if they are never rendered from
+  far away. Disabling this can reduce output file size and speed up importing.
+  See :ref:`doc_mesh_lod` for more information.
+- **Create Shadow Meshes:** If checked, enables the generation of
+  shadow meshes on import. This optimizes shadow rendering without reducing
+  quality by welding vertices together when possible. This in turn reduces the
+  memory bandwidth required to render shadows. Shadow mesh generation currently
+  doesn't support using a lower detail level than the source mesh (but shadow
+  rendering will make use of LODs when relevant).
+- **Light Baking:** Configures the meshes'
+  :ref:`global illumination mode <class_GeometryInstance3D_property_gi_mode>`
+  in the 3D scene. If set to **Static Lightmaps**, sets the meshes' GI mode to
+  **Static** and generates UV2 on import for :ref:`lightmap baking <doc_using_lightmap_gi>`.
+- **Lightmap Texel Size:** Only visible if **Light Baking** is set to **Static
+  Lightmaps**. Controls the size of each texel on the baked lightmap. A smaller
+  value results in more precise lightmaps, at the cost of larger lightmap sizes
+  and longer bake times.
+
+**Skins**
+
+- **Use Named Skins:** If checked, use named skins for animation.
```

Comment:
I don't understand what named skins are for exactly. I remember this having to do with animation reuse on different unrelated models. Can someone help me fill this description?

Reply:
Okay, so let's start with explaining what a Skin object is. Actually let's go back more and explain what the role of a Skeleton3D node is, and how it relates to Mesh rendering.

The MeshInstance3D node contains three properties of relevance: a skeleton node path, pointing to the Skeleton3D node, usually "..", a mesh, and a skin:

- The Skeleton3D node contains a list of bones with names, their pose and rest, a name and a parent bone.
- The mesh is all of the raw vertex data needed to display a mesh. In terms of the mesh, it knows how vertices weight painted, and uses some internal numbering often imported from a Digital Content Creation (DCC) tool such as Blender.
- The skin contains the information necessary to bind this mesh onto this Skeleton3D. For every one of the internal bone ids chosen by the DCC tool, it contains two things. Firstly, a Matrix known as the Bind Pose Matrix, Inverse Bind Matrix, or IBM for short. Secondly, the Skin contains each bone's name (if the Use Named Binds setting is on by default), or the bone's index within the Skeleton3D list (if Use Named Binds is turned off).

Together, this information is enough to tell Godot how to use the bone poses in the Skeleton3D node to render the mesh from each MeshInstance3D. Note that each MeshInstance3D may share binds, as is common in models exported from Blender, or each MeshInstance3D may use a separate Skin object, as is common in models exported from other tools such as Maya.

Reply:
Thanks a lot! I've added this information to the page :slightly_smiling_face: